src/main/java/in/projecteka/devservice/clients/ServiceAuthenticationClient.java

```diff
@@ -107,6 +110,27 @@ private BridgeServiceRequest bridgeServiceRequest(String id, String name, Servic
                 .then();
     }
 
+    public Mono<SupportBridgeResponse> getClientIdAndSecret(SupportBridgeRequest request, String token) {
```

Comment:
Aren't we generating a new one here? If yes then let's rename this to generateClientIdAndSecred

Reply:
Done